charts/apisix/templates/configmap.yaml

```diff
@@ -226,6 +226,13 @@ data:
 
     {{- if .Values.customPlugins.enabled }}
     plugin_attr:
+      {{- if .Values.serviceMonitor.enabled }}
+      prometheus:
+        export_addr:
+          ip: 0.0.0.0
```

Comment:
It's dangerous and if APISIX instances are exposed to the Internet, the metrics data can be fetched by others.

Reply:
So how shall we expose metrics from APISIX gateway, any better approach?

Reply:
Maybe we can use the PodMonitor to fetch metrics, then we don't have to export the port to the service, and at least accessors outside the cluster cannot snoop the metrics.

Reply:
Got your point, while if you check the codes, will find that no ports were exposed in `Service`, we only expose one essential container port in `Deployment`, and in this way `ServiceMonitor` could still work well.

This [issue](https://github.com/prometheus-operator/prometheus-operator/issues/3119#issuecomment-631409690) may echo my understanding: "It will scrape all pods behind the service, because the `Service` maintains an `Endpoints` object. That's where Prometheus discovers the targets from."

Reply:
Is the endpoint port defined on ServiceMonitor is a container port? If so, I think is OK.

Reply:
Yes, that's container port, not service port. :)

Reply:
OK.